docs/eog/eog-in-practice/cw21/event-project-stage/eps-risk-management.rst

```diff
@@ -0,0 +1,185 @@
+- Damage to the main stakeholder’s reputation
+
+As we demonstrated through CW20, the event goals and objectives can be achieved through a virtual event.
+To ensure that the :ref:`cw21-fs-goals-and-objectives` aligned with the main stakeholder goals, the Director was consulted during the :ref:`cw21-feasibility-stage`, and both the Director and Research Director were invited to participate on the :ref:`CW21 Steering Committee <cw21-eps-committees>`.
+
+Running CW20 virtually at the start of the pandemic led to the Institute being internationally recognised and regularly consulted on virtual event organisation and community building. 
+70% of CW20 participants who gave feedback stated that "CW20 is the best [online] event I have attended".
+
+.. figure:: ../img/CW20_participant_feedback1.png
+  :alt: CW20 Participant Feedback on how the event compared with other online events 
+  
+  Feedback from CW20 participants on how the event compared with other online events
+
+Due to the attention and care that we put into improving and building off of what we achieved during CW20, we had similar feedback after CW21 when compared with the past year of virtual events. 
+
+.. figure:: ../img/CW21_participant_feedback1.png
+  :alt: CW21 Participant Feedback on how the event compared with other online events 
+  
+  Feedback from CW21 participants on how the event compared with other online events
+
+
+Other stakeholders 
+--------------------
+
+Attendees
+^^^^^^^^^^^^
+
+The risks identified regarding the attendees include:
+
+- Not enough attendees or many dropping out right before the event
+- Too many attendees
+
+The strategy we took to mitigate these risks included:
+
+- Keep a registration fee in place for a virtual event to minimise no-shows
+- Limit the number of registrations to 150 so that we could manage the interactive sessions and have a more intimate event
+
+Within the CW20 feedback form, we asked participants how they would prefer us to organise CW in the future (in-person or online).
+Half of respondents said that they didn't mind how it was run, and 33% said that they prefer an in-person event.
+
+.. figure:: ../img/CW20_participant_feedback2.png
+  :alt: CW20 Participant Feedback on how to organise the event in the future
+  
+  Feedback from CW20 participants on how to organise the event in the future 
+
+However, as the effects of the pandemic were ongoing, we decided that CW21 would take place online.
+As the vaccination programmes advance and things begin to open up, there is pressure and expectation for hybrid events. 
+We asked CW21 on the feedback form to rank how they prefer future CWs to be organised: in-person, online, hybrid or don't mind, the results of which are displayed below:
+
+.. figure:: ../img/CW21_participant_feedback2.png
+  :alt: CW21 Participant Feedback on how to organise the event in the future 
+  
+  Feedback from CW21 participants on how to organise the event in the future
+
```

Comment:
```suggestion

Opinions were more or less evenly split between the different options, and we believe this may be due to ongoing uncertainty about the pandemic.
```